src/components/organisms/DisplayWalletBalance.tsx

```diff
@@ -0,0 +1,34 @@
+import { useEffect, useState } from "react";
+import { getWalletBalance } from "../../lib/brokerHandler";
+import TextDisplay from "../atoms/TextDisplay";
+
+const DisplayWalletBalance = () => {
+  const [walletBalance, setWalletBalance] = useState<string>("");
```

Comment:
Change initial value to something that informs about the state. For example "(checking...)"